site/docs/api-quickstart.md

```diff
@@ -69,7 +69,7 @@ logsDF.write
 !!! Warning
     Hadoop tables shouldn't be used with file systems that do not support atomic rename. Iceberg relies on rename to synchronize concurrent commits for directory tables.
 
-### Tables in Spark
+### Tables in Apache Spark
```

Comment:
I find it distracting to add "Apache" everywhere, when it is clear that "Spark" refers to Apache Spark. I think we should use "Apache Spark" at most once (to meet trademark guidelines), and not in a section heading. Same with "Apache Avro" because the reference is clear.

Reply:
Removed a couple, let me know if it is still too much